### DISCUSSION TITLE
Fat Cell: Eats adjacent food and stores food as lifetime.

### DISCUSSION THREAD

**Discussion by theomgdev** [MaxRobinsonTheGreat/EvolutionSimulatorV2#76] [Ideas and Suggestions]
#74 
Fat - Light yellow, eats food in directly adjacent cells but uses food to increase lifetime instead of reproduction.

(I used my bait-cell as a template. Just ignore old commits.)

**Just let them starve to see the magic!**

**Comment by Coolboy-10** [comment]
👍
